source/jormungandr/tests/chaos_disruptions_tests.py

```diff
@@ -89,6 +90,9 @@ def test_disruption_on_stop_area_b(self):
         assert any(d['disruption_id'] == 'bob_the_disruption' for d in disruptions)
         assert disruptions[0]['cause'] == 'CauseTest'
         assert 'category' not in disruptions[0]
+        assert len(disruptions[0]['tags']) == 2
+        assert 'metro' in disruptions[0]['tags']
```

Comment:
You only test that tags are outputed, not the filtering if I understand correctly.
Can you add a test on filtering as it's the main feature of this PR?

Reply:
:heavy_check_mark:  Done